docs/modules/reference/pages/service-assurance/monitors/DeviceConfigMonitor.adoc

```diff
@@ -34,17 +34,20 @@ Don't include `.dcb` extension when specifying the file name in metadata.
 | username
 | Username for SSH login. 
 Overwrite in metadata as `dcb:username`.
+Note that you can use Secure Credentials Vault (SCV) metadata.
 | admin
 
 | password
 | Password for SSH login. 
 Overwrite in metadata as `dcb:password`.
+Note that you can use Secure Credentials Vault (SCV) metadata.
 | admin
 
 3+| *Optional*
 
 | config-type
 | If device has multiple configurations, specify the config-type with each service.
+The config-type and service name suffix should match (kebab-case for Service name, camelCase for config-type).
```

Comment:
I have never heard of kebab case. Thanks! I love learning new things. 
```suggestion
The config-type and service name suffix should match (kebab case for service name, camelCase for config-type).
```

Reply:
I used `kebab-case` and `camelCase` with that specific formatting to match how they look, but maybe that isn't clear!

Reply:
Doh! Yes, kebab-case would be correct since it is showing how they look. Over-correction on my part.